bundles/core/org.eclipse.smarthome.core.transform/src/main/java/org/eclipse/smarthome/core/transform/TransformationHelper.java

```diff
@@ -81,6 +81,19 @@ public static TransformationService getTransformationService(BundleContext conte
      * @return the result of the transformation. If no transformation was done, the state is returned
      */
     public static String transform(BundleContext context, String stateDescPattern, String state) {
+        return transform(context, stateDescPattern, state, null);
+    }
+
+    /**
+     * Transforms a state string using transformation functions within a given pattern.
+     *
+     * @param context a valid bundle context, required for accessing the services
+     * @param stateDescPattern the pattern that contains the transformation instructions
+     * @param state the state to be formatted before being passed into the transformation function
+     * @param itemName the name if the item whose state should be converted or null
```

Comment:
The sentence is a little bit strange and it should indicate that itemName is optional.